spooky-oysters/bangazon
# migrations for Computer and ComputerEmployee models

## Migration for Computer.cs and ComputerEmployee.cs models

## Description of Proposed Changes

- Migrations for updating the database with Computer and ComputerEmployee tables

## Steps to Test

- Remove local database file

```sh
git fetch --all
git checkout dr-migrateComputerModels
dotnet ef database update
```
- Open the project database file in DB Browser and check for the Computer and ComputerEmployee tables 
- Check that Computer has a nullable `DecommisionDate` row
- Check that ComputerEmployee has a nullable `DateRemoved` row
- Check that Order has a nullable `CompletedDate` row

## Impacted Areas in Application

* Migrations
* Local and global database

## Mentions @username

Tag users that need to review this code
